aws/aws.go

```diff
@@ -85,40 +85,43 @@ func HasServiceName(r interface {
 }
 
 // S3API aws API
-type S3API s3iface.S3API
+type S3API = s3iface.S3API
```

Comment:
FME (for my edification): did you manually make those changes, or was it via an automated tool?

Reply:
I made these changes. New types weren't needed, only aliases.